addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -334,7 +334,9 @@ def _process_step_forward_operator(self, mail_channel):
         if mail_channel.livechat_channel_id:
             human_operator = mail_channel.livechat_channel_id._get_random_operator()
 
-        if human_operator:
+        # handle edge case where we found yourself as available operator -> don't do anything
+        # it will act as if no-one is available (which is fine)
+        if human_operator and human_operator != self.env.user:
```

Comment:
Shouldn't we exclude ourself from get_random_operator in that case ?

Reply:
That would technically be the "good" fix to do.

Here I went for the "cheap" one as it requires changing the API.
(Or calling _get_random_operator multiple times but that's really bad design).

And honestly I don't really feel like this is worth it considering this is a very edge case, I'm ok with it just "not crashing the script".